10x/count/count.wdl

```diff
@@ -0,0 +1,1427 @@
+# stage SETUP_CHUNKS(
```

Comment:
I understand why you have this here, but it may be best for now to just reference the mro rather than pasting it here. For example, could you say "See SETUP_CHUNKS in cellranger/mro/_common_stages.mro" and/or link directly to it like this?https://github.com/10XGenomics/cellranger/blob/e6517cae10b030973968dec91e09710063297fde/mro/_common_stages.mro#L35

Reply:
Good idea, I've replaced all the MRO with github links.